sda/host.go

```diff
@@ -612,3 +617,63 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+//Received gives packets received
+func Received(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Rx()
+	}
+	return a
```

Comment:
Ah, OK I see. Thanks. Could you add a very short comment (inside the function). Sth like you wrote `// grabbing a single part of a map without a key` Because it doesn't really look as if this was on purpose.